### PR TITLE
Import history of CppCheck

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -20745,6 +20790,7 @@ simmilar->similar
 simpified->simplified
 simplei->simply
 simpley->simply
+simplication->simplification
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: Does exist.
https://en.wiktionary.org/wiki/simplication

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -24620,6 +24682,7 @@ withe->with
 witheld->withheld
 withh->with
 withih->within
+withinfuntion->withinfunction
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: two words?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -23556,6 +23612,7 @@ unknows->unknowns, unknows,
 unknwoing->unknowing
 unknwoingly->unknowingly
 unknwon->unknown
+unknwonlanguage->unknownlanguage
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: two words?

**Comment by luzpaz** [comment]
bump

**Comment by bl-ue** [comment]
Wow, really cool.  I'll take this, refresh it, and push it to completion.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -6373,6 +6389,7 @@ coyp->copy
 coypright->copyright
 coyright->copyright
 cpation->caption
+cpcheck->cppcheck
 cpoy->coy, copy,
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:6393]:
```suggestion
cpoy->copy, coy,
```

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (bl-ue) [reply codespell_lib/data/dictionary.txt:20793]: I'll move it to rare.

**Comment by bl-ue** [comment]
Yay! 🎉

**Review by bl-ue — APPROVED** [review]

**Pull request merged**